airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java

```diff
@@ -127,20 +122,12 @@
    *
```

Comment:
There are many changes in this file since I did a fair amount of refactoring here for readability.

Main changes are adding a `namespace` field to the `retrieveRecords` and `retrieveNormalizedRecords` functions.

I rearrange the test methods so it's
1) Get
2) Check(s)
3) Syncs
4) Incremental
5) Normalisation
6) Namespace

And pushed all the helper functions down.

Reply:
the diff on this file is hard to parse. i'm assuming it is just 1. a rearrange, 2 adding the defaultNamespace method, and 3 adding this test:  source does not specify a schema. lmk if there's another change in here i should be looking for.

Reply:
aye apologies. I should have broken the refactoring changes into another commit. Will be better next time.

yes you were looked at all the right bits!